src/Core/tests/DeviceTests/Handlers/Editor/EditorHandlerTests.cs

```diff
@@ -21,6 +21,22 @@ public async Task TextInitializesCorrectly()
 			await ValidatePropertyInitValue(editor, () => editor.Text, GetNativeText, editor.Text);
 		}
 
+		[Fact(DisplayName = "Text Initializes Correctly when Keyboard is Before Text")]
+		public async Task TextInitializesCorrectlyWhenKeyboardIsBeforeText()
```

Comment:
With all these tests, I'm confused by what you mean by "Before Text". Isn't the text being mapped/initialized first?

Reply:
fixed.